synapseclient/client.py

```diff
@@ -639,21 +638,19 @@ def invalidateAPIKey(self):
         if self._is_logged_in():
             self.restDELETE("/secretKey", endpoint=self.authEndpoint)
 
-    @memoize
-    def getUserProfile(self, id=None, sessionToken=None, refresh=False):
```

Comment:
Is there a comparable parameter to refresh with the new function you are using?

Reply:
I searched and I can't see a parameter to refresh by using @lru_cache. Python also has `cache_clear()` and `cache_info()`. But I think the purpose of refresh is to grab the new data instead of using the cached result, which I think is different from clearing cache completely.